packages/nextjs/components/bgChain/BridgeToBGChain.tsx

```diff
@@ -0,0 +1,104 @@
+import { useState } from "react";
+import { Address, Balance, InputBase, getParsedError } from "../scaffold-eth";
+import { useConnectModal } from "@rainbow-me/rainbowkit";
+import { parseEther } from "viem";
+import { sepolia, useAccount, useNetwork, useSwitchNetwork, useWalletClient } from "wagmi";
+import { ArrowsRightLeftIcon, BanknotesIcon } from "@heroicons/react/24/outline";
+import { useNetworkColor, useTransactor } from "~~/hooks/scaffold-eth";
+import { bgChainL2 } from "~~/scaffold.config";
+import { notification } from "~~/utils/scaffold-eth";
+
+const BRIDGE_CONTRACT_ADDRESS = "0x729593813494173Ce5d57743692c1E022425765f";
+
+const BridgeToBGChain = () => {
+  const [loading, setLoading] = useState(false);
+  const [sendValue, setSendValue] = useState("");
+  const { address: connectedAddress, isConnected, isConnecting } = useAccount();
+  const { data: walletClient } = useWalletClient();
+  const { chain: connectedChain } = useNetwork();
+  const sepoliaColor = useNetworkColor(sepolia);
+  const { switchNetwork } = useSwitchNetwork();
+  const { openConnectModal } = useConnectModal();
+
+  const bridgeTxn = useTransactor(walletClient);
+
+  const sendETH = async () => {
+    if (!connectedAddress) {
+      return;
+    }
+    try {
+      setLoading(true);
+      await bridgeTxn({
+        to: BRIDGE_CONTRACT_ADDRESS,
+        value: parseEther(sendValue as `${number}`),
+        account: connectedAddress,
+        chain: sepolia,
+      });
+      setLoading(false);
+      setSendValue("");
+    } catch (error) {
+      const parsedError = getParsedError(error);
+      console.error("⚡️ ~ file: Faucet.tsx:sendETH ~ error", error);
```

Comment:
Sorry for the nitpick 😊 => `console.error("⚡️ ~ file: BridgeToBGChain.tsx:sendETH ~ error", error);`

Reply:
Tysm !! removed it at [30f4ad7](https://github.com/BuidlGuidl/l2.buidlguidl.com/pull/5/commits/30f4ad7300dcd3a5e54a2898f7be6901c8912675) 🙈 ..We can add it while debugging locally